pkg/identityserver/store/gateway.go

```diff
@@ -116,6 +121,15 @@ var gatewayPBSetters = map[string]func(*ttnpb.Gateway, *Gateway){
 			pb.Antennas[i] = antenna.toPB()
 		}
 	},
+	lbsLNSSecretField: func(pb *ttnpb.Gateway, gtw *Gateway) {
+		blocks := bytes.SplitN(gtw.LBSLNSSecret, lbsLNSSecretSeparator, -1)
```

Comment:
That -1 doesn't look right. I think it should be 2 instead.

Reply:
Ah yeah. The plan was to allow `:` in the secret indeed.